packages/cozy-client/src/store/queries.spec.js

```diff
@@ -22,15 +22,40 @@ describe('queries reducer', () => {
   })
 
   it('should init correctly', () => {
+    const qdef = new QueryDefinition({
+      doctype: 'io.cozy.todos'
+    })
+    applyAction(initQuery('a', qdef))
+    expect(state).toMatchSnapshot()
+  })
+
+  test('init should have no effect after receiveQueryResult', () => {
```

Comment:
Very minor, but it's strange to use `test` instead of `it` all of a sudden.

Reply:
It's because the sentence cannot be read with "it" :)